CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 
 ### Changed
+- BREAKING: Introduced proper error handling for `parse_cli_properties`, `parse_env_properties` and `parse_file_properties` ([#282]).
```

Comment:
Isn't `transform_all_roles_to_config` the public part of the API that changed here?

Reply:
True, fixed.